spec/classes/dns_init_spec.rb

```diff
@@ -65,6 +65,15 @@
         end
       end
 
+      let(:defaultzonepath) do
+        case facts[:os]['family']
+        when 'Debian'
+          "#{etc_directory}/named.conf.default-zones"
+        when 'RedHat'
+          'unmanaged'
```

Comment:
I'd leave this out and only deal with absolute paths (or nil) in tests.
```suggestion
```

Reply:
Done, updated.